README.md

```diff
@@ -1,7 +1,7 @@
 # Lucupy
 
 This package contains data structures and functions to support all the microservices that make up the Schedule app
-for the Gemini Program Plataform (and other auxiliary services such as Env and Resource)
+for the Gemini Program Platform (and other auxiliary services such as Env and Resource)
```

Comment:
Ugh my spanish leaking out.

Reply:
🤣 Well, I didn't know the Spanish word for platform, so it was a learning opportunity.